src/main/java/ir/sahab/neor/kafka/reader/KafkaProtoParquetWriter.java

```diff
@@ -132,12 +114,12 @@ private KafkaProtoParquetWriter(Builder<T> builder) {
         protoClass = builder.protoClass;
         parser = builder.parser;
 
-        consumerConfig = new HashMap<>(builder.consumerConfig);
-        consumerConfig.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, false);
-        consumerConfig.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG,
-                ByteArrayDeserializer.class);
-        consumerConfig.computeIfAbsent(ConsumerConfig.GROUP_ID_CONFIG,
-            key -> "KafkaProtoParquetWriter-" + instanceName);
+        // Kafka consumer config
+        Properties kafkaConfig = new Properties();
```

Comment:
Rename kafkaConfig to kafkaConsumerConfig

Reply:
done